source/whats-new.txt

```diff
@@ -73,6 +74,22 @@ New versions of the {+driver-short+} can introduce the following types of breaki
      - Drops support for platform A, compiler B, or architecture C.
 
 .. _upcoming-breaking-changes:
+.. _version-4.0:
+
+What's New in 4.0
+-----------------
+
+.. warning:: Breaking Changes
+
+   The v4.0 driver introduces the following breaking changes:
+   
+   - Drops support for the Boost and MNMLSTC/core external polyfill libraries. You can no
```

Comment:
> You can no longer

Second person perspective seems odd. Is this deliberate?

Reply:
We use second person in the docs, as per the [style guide](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/)